### ISSUE TITLE
a11y: Fix keyboard navigation and focus display for cluster appearance configuration

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12781]
a11y Issue: 119, 137

**Button on Cluster Dashboard**

The cluster badge configuration button (see below) does not show keyboard focus and does not response to enter/space to activate.

![image](https://github.com/user-attachments/assets/18337dfc-bb56-480b-b87b-066d7eded45b)

**Cluster Appearance Dialog**

Tooltip content not accessible with a keyboard.

![image](https://github.com/user-attachments/assets/79f60ae2-4de4-48de-b859-20930937e0a7)

**Comment by yonasberhe23** [comment]
Needs manual validation

**Comment by IsaSih** [comment]
Works as expected. Tests pass in 
Rancher | v2.11-a1bda936c809e0394520c9d40d2998529223401e-head
-- | --
Dashboard | master c987984
Rancher CLI | v2.11.0-rc.1

**Comment by IsaSih** [comment]
Automation tracked by https://github.com/rancher/qa-tasks/issues/1639
